app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -258,7 +258,7 @@ class User private constructor(
                 val matches = get("matches") as? List<String>
                 val likes = get("likes") as? List<String>
                 val description = getString("description")!!
-                val ageRange = get("ageRange") as? List<Long>
+                val ageRange = get("ageRange") as? List<Int>
```

Comment:
if the test pass i guess it is ok but for know the checks fail so ...

Reply:
Yep, my bad I meant to ask if it was a change from Melissa or if I had changed it by mistake (I'll change it back)

Reply:
I did change that, because the get("ageRange") returns as a List<Long> and 'as List<Int>' doesn't do anything on a List<Long>. So I put List<Long> to make it clearer, since later in the code I manually transform ageRange into a List<Int> ^^